resources/assets/js/app/views/users/userCollection.js

```diff
@@ -0,0 +1,8 @@
+var Marionette = require('backbone.marionette');
+
+module.exports = Marionette.CollectionView.extend({
+    template: '#users',
```

Comment:
First of all because collectionView doesn't use template) second, template names doesn't contain #